web_demo/index.html

```diff
@@ -48,9 +48,10 @@
             margin-left: auto;
             display: block;
             position: absolute;
-            top: 0%;
-            left: 50%;
-            transform: translate(-50%, 0%);
+            top: 0;
```

Comment:
We should have updated the comment for the `canvas` block as well. I don't think it is accurate anymore.